frameworks/Kotlin/ktor/benchmark_config.json

```diff
@@ -2,15 +2,15 @@
   "framework": "ktor",
   "tests": [
     {
-      "default": {
+      "netty": {
```

Comment:
so if I have one `"default"` and one `"jetty"` then what test will be started if I specify no test name? (like `tfb -m benchmark`) Both or only the default one?